includes/class-wcs-import-admin.php

```diff
@@ -518,7 +518,7 @@ public function save_mapping() {
 		}
 
 		foreach ( $mapping_rules as $key => $value ) {
-			if ( ! empty( $value ) && is_array( $mapped_fields[ $value ] ) ) {
+			if ( ! empty( $value ) && isset( $mapped_fields[ $value ] ) && is_array( $mapped_fields[ $value ] ) ) {
```

Comment:
Not related to this PR - please revert here and propose the change in a separate one.
```suggestion
			if ( ! empty( $value ) && is_array( $mapped_fields[ $value ] ) ) {
```